app/controllers/concerns/edge_cache_safety_check.rb

```diff
@@ -3,8 +3,6 @@
 module EdgeCacheSafetyCheck
   extend ActiveSupport::Concern
 
-  CANNOT_USE_CURRENT_USER = "You may not use current_user in this cached code path.".freeze
```

Comment:
This is not directly user-facing as it's only enabled in local development. Given we generally require people to reasonably comfortable in English to contribute I'm not sure there's much value in translating this.